scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -51,3 +104,56 @@ SOP-8_6.62x9.15mm_P2.54mm:
   pitch: 2.54
   num_pins_x: 0
   num_pins_y: 4
+
+SOP-16_4.4.55x10.3mm_P1.27mm:
+  size_source: 'https://toshiba.semicon-storage.com/info/docget.jsp?did=12855&prodName=TLP290-4'
+  body_size_x:
+    nominal: 4.55
+    tolerance: [-0.15, 0.25]
+  body_size_y:
+    nominal: 10.3
+    tolerance: [-0.15, 0.25]
+  overall_height:
+    nominal: 2.1
+    tolerance: 0.14
+
+  overall_size_x:
+    nominal: 7
+    tolerance: 0.4
+  lead_width:
+    nominal: 0.4
+    tolerance: 0.1
+  lead_len:
+    minimum: 0.5
+    maximum: 0.5
+    # Only minimum given in the datasheet.
+    # With 0 tolerance we already get longer pads than suggested by the datasheet -> should be ok
+
+  pitch: 1.27
+  num_pins_x: 0
+  num_pins_y: 8
+
+SOP-18_7.0x12.5mm_P1.27mm:
```

Comment:
Since the Toshiba datasheet has many similarly-named packages and is laid out, in my opinion, poorly, could you capture the Toshiba package name for which this footprint is designed (`SOP18-P-375-1.27`)?